### PR TITLE
NCAS-3762 #comment update docdev with the latest API spec

### PR DESCRIPTION
**Pull request by burtonquantil**
Push the changes tested by QA on another environment to docdev.quantil.com

### CODE REVIEW

File: en/apispec/openapi.json [hunk 1]
```diff
@@ -31833,6 +31833,126 @@
...
+                  "properties": {
+                    "ipDetails": {
+                      "type": "array",
+                      "description": "Describes an IP address.",
+                      "items": {
+                        "type": "object",
+                        "properties": {
+                          "ip": {
+                            "type": "string",
+                            "description": "An IPv4 or IPv6 address"
```
Comment (chiraags) [inline en/apispec/openapi.json:31859]: Should we include a period after the statement, similar to other descriptions?

File: zh/apispec/openapi.json [hunk 2]
```diff
@@ -31833,6 +31833,126 @@
...
+                  "properties": {
+                    "ipDetails": {
+                      "type": "array",
+                      "description": "Describes an IP address.",
+                      "items": {
+                        "type": "object",
+                        "properties": {
+                          "ip": {
+                            "type": "string",
+                            "description": "An IPv4 or IPv6 address"
```
Comment (chiraags) [inline zh/apispec/openapi.json:31859]: Same question as above.

**Review by chiraags — APPROVED** [review]
Good to merge.

**Comment by burtonquantil** [comment]
Thanks @chiraags for reviewing.   I'll add the periods when I update the API spec to cover sprint 144.

**Pull request merged**